### PR TITLE
Add avatar to user model #5

### PR DESCRIPTION
**Pull request by kaniak274**

### CODE REVIEW

File: yumljira/apps/users/test_factories.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from django.contrib.auth import get_user_model
+from django.core.files.uploadedfile import SimpleUploadedFile
 
 import factory
```
Comment (KamilSupera) [inline yumljira/apps/users/test_factories.py:4]: It should be first import?

File: yumljira/apps/users/tests.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+from django.conf import settings
+from django.core.files.base import File
+from django.urls import reverse
+
+from faker import Faker
+
+import pytest
```
Comment (KamilSupera) [inline yumljira/apps/users/tests.py:7]: Same ^^^

File: yumljira/apps/users/test_factories.py (hunk 1 above)
Comment (kaniak274) [reply yumljira/apps/users/test_factories.py:4]: Nah this is not python package

File: yumljira/apps/users/tests.py (hunk 2 above)
Comment (kaniak274) [reply yumljira/apps/users/tests.py:7]: Nah this is not python package

**Pull request merged**